### DISCUSSION TITLE
Adonde se debe subir en GitHub la Practica de Abogabot

### DISCUSSION THREAD

**Discussion by 79155637** [Launch-X-Latam/MisionFrontEnd#205] [Q&A]
Buen día señores LaunchX-FrontEnd: primero agradecerles por las clases de FrontEnd de Rodrigo Martínez, pero he visto todos los videos y estoy haciendo la practica de Abogabot correspondiente a la introducción y siempre Rodrigo dijo esperáramos a que el diera las instrucciones, y de repente dice que la subamos a GitHub, la pregunta es: bajo que parámetros lo subo GitHub ?? o tienen algún instructivo en su repositorio de FrontEnd de LaunchX, por favor me pueden colaborar, mil gracias de antemano por su ayuda

**Comment by EsthebanDiego** [comment]
Yo tengo entendido es que cada uno crea un repositorio en su cuenta de GitHub, y una vez ellos habiliten el formulario, nosotros debemos poner el link de nuestro repositorio

**Comment by 79155637** [comment]
Gracias, pero eso hace tiempo lo hice y tengo mi Github registrado en LATAM
X.
La pregunta es exactamente donde se deben subir las practicas de FrontEnd.
Si lo sabe exactamente, le agradecería la información

David Eduardo Velandia Laverde

**Reply by EsthebanDiego** [reply]
aun no crean el formulario para subir los archivos, he visto mensajes que indican que se creara un formulario hacia los últimos días de Octubre, pero aún no hay nada en concreto.  de momento la única instrucción es la de crear el repositorio
